float64 epoch     # Time
float64 roll      # Rotation about the X axis (rads)
float64 pitch     # Rotation about the Y axis (rads)
float64 yaw       # Rotation about the Z axis (rads)